web/styles/compose.css

```diff
@@ -277,19 +277,37 @@
     border-radius: 5px;
     border: 1px solid;
     display: flex;
-    align-items: center;
+    align-items: flex-start;
     justify-content: space-between;
     font-size: 15px;
     line-height: 18px;
 
+    .main-view-banner-elements-wrapper {
+        display: flex;
+        width: 100%;
+        justify-content: space-between;
+
+        @media (width < $md_min) {
+            flex-direction: column;
+            align-items: flex-start;
+        }
+    }
+
     & p {
         margin: 0; /* override bootstrap */
         /* 5px right padding + 10px left-margin of the neighbouring button will match the left padding */
         padding: 8px 5px 8px 15px;
     }
 
-    .banner_content {
-        flex-grow: 1;
+    @media (width > $md_min) {
```

Comment:
I think we should add an `=` here.
> @media (width >= $md_min)

Currently, at 768px, the close button is not properly aligned.
![Screenshot from 2023-06-14 22-50-23](https://github.com/zulip/zulip/assets/56781761/28a90d3a-43e9-4fbc-9858-5153d1f27ad8)

Reply:
@prakhar1144, Updated the PR to tweak this, this PR is ready for review now.